R/recipe.R

```diff
@@ -100,19 +103,13 @@ recipe.data.frame <-
            roles = NULL) {
     if (!is.null(formula)) {
       if (!is.null(vars)) {
-        rlang::abort(
-          paste0(
-            "This `vars` specification will be ignored ",
-            "when a formula is used"
-          )
+        cli::cli_abort(
+          "The {.arg vars} argument will be ignored when a formula is used."
         )
       }
       if (!is.null(roles)) {
-        rlang::abort(
-          paste0(
-            "This `roles` specification will be ignored ",
-            "when a formula is used"
-          )
+        cli::cli_abort(
+          "This {.arg roles} argument will be ignored when a formula is used."
```

Comment:
```suggestion
          "The {.arg roles} argument will be ignored when a formula is used."
```

Maybe? For consistency with the message above.